src/pages/dormitory.tsx

```diff
@@ -59,6 +59,8 @@ const Dormitory: React.FC<DormitoryProps> = ({ setShowingHeader }) => {
 
         <main className="section pt-0 pb-5">
           <div className="container character-container is-max-desktop pt-1 pb-6">
+            <hr />
+
             <div className="columns is-multiline">
```

Comment:
HTMLの構造的に一番上にhrが来ているのはおかしいかもと思いました（SEO的にマイナスかも？）
ここは大きめの空白が開いていて上下の意味が異なることがなんとなくわかる気がするので、線がなくてもいいかもです！

```suggestion
          <div className="container character-container is-max-desktop pt-1 pb-6">
            <div className="columns is-multiline">
```